content/en/docs/concepts/storage/storage-capacity.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- jsafrane
+- saad-ali
+- msau42
+- xing-yang
+- pohly
+title: Storage Capacity
+content_type: concept
+weight: 45
+---
+
+<!-- overview -->
+
+Storage capacity is limited and may vary depending on the node on
+which a pod runs: network-attached storage might not be accessible by
+all nodes, or storage is local to a node to begin with.
+
+This page describes how Kubernetes keeps track of storage capacity and
+how the scheduler uses that information to schedule Pods onto nodes
+that have access to enough storage capacity for the remaining missing
+volumes. Without storage capacity tracking, it is random whether the
+selected node can run the Pod and multiple scheduling retries may be
+needed.
+
+<!-- body -->
+
+
+## Enabling the feature
+
+Storage capacity tracking is an *alpha feature* and only enabled when
+the `CSIStorageCapacity` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled. A quick check
+whether a Kubernetes cluster supports the feature is to list
+`CSIStorageCapacity` objects with:
+```shell
+kubectl get csistoragecapacities --all-namespaces
+```
+
+If supported, the response will a list of objects or:
+```
+No resources found
+```
+
+If not supported, this error is printed instead:
+```
+error: the server doesn't have a resource type "csistoragecapacities"
+```
+
+In addition to enabling the feature in the cluster, a [CSI
+driver](/docs/concepts/storage/volumes/#csi) deployment also has to
```

Comment:
consider avoiding the term “deployment” as it could mislead readers.

Reply:
I know what you mean.

Strictly speaking, the support for the feature really is in the deployment (yaml files, operator, etc) and not just in the CSI driver itself. But I think I can leave out this fine detail and drop the term.